cmd/vicadmin/vicadm.go

```diff
@@ -85,6 +87,21 @@ func init() {
 	// This is only applicable for containers hosted under the VCH VM folder
 	// This will not function for vSAN
 	flag.StringVar(&config.vmPath, "vm-path", "", "Docker vm path")
+	authenticatorMap = map[string]func(http.HandlerFunc) http.HandlerFunc{
+		// add new authentication methods here
+		"basic": basicHTTPAuthenticator,
+	}
+
+	var validTypes bytes.Buffer
+	for key := range authenticatorMap {
+		validTypes.WriteString(key)
+		validTypes.WriteString(" ")
+	}
+
+	// generate help text so it doesn't need to be updated later
+	authTypeHelp := "Set the authentication type to use. Valid types: " + validTypes.String()
+	flag.StringVar(&config.authenticator, "auth-type", "basic", authTypeHelp)
```

Comment:
Instead of authTypeHelp, "Authentication type: " + strings.Join(authTypes, "|")
We should default to no authentication for now, only using the in memory credentials for _test.go - so you can also just omit the flag until we have an authenticator that can be used outside of testing.